code/client/munkiimport

```diff
@@ -691,12 +691,17 @@ def cleanupAndExit(exitcode):
     exit(exitcode or result)
 
 
-def pref(prefname):
-    """Returns a preference for prefname"""
+def pref(prefname, value=None):
+    """Returns or sets preference for prefname"""
     try:
         _prefs = FoundationPlist.readPlist(PREFSPATH)
     except FoundationPlist.NSPropertyListSerializationException:
         return None
+
+    if value:
+        _prefs[prefname] = value
+        return FoundationPlist.writePlist(_prefs, PREFSPATH)
+
     if prefname in _prefs:
```

Comment:
:) When you get a value, it is returned, when you set it, it is set and nothing is returned. Could also always just return the value, but I don't need to get back what I just set.